### PR TITLE
Improve accuracy and completeness of results

### PR DESCRIPTION
**Pull request by paolo-github**
Two changes to improve:
- accuracy of results by increasing the number of repetitions
- completeness of results, by reporting full statistics instead of just averages (this improvement is actually implemented in the core S suite, this change only modifies comments consistently)

### CODE REVIEW

**Comment by mwasilew** [comment]
@paolo-github After I merged #122 your patch needs rebasing. Could you take a look?

**Comment by paolo-github** [comment]
Rebased and made a new PR.

Thanks,
Paolo

**Comment by mwasilew** [comment]
Looks like #123 and #124 are identical. Please close one. I added comments to #124.

**Comment by Algodev-github** [comment]
> Looks like #123 and #124 are identical. Please close one. I added comments to #124.

Trying to, but not finding how exactly (never done before, sorry)

**Comment by mwasilew** [comment]
Just let me know which one to close and I'll do that.

File: automated/linux/ssuite/run-bench.sh [hunk 1]
```diff
@@ -173,18 +173,18 @@ run_test() {
 	rm -f ${HOME_DIR}/.S-config.sh
 
 	cd "$S_PATH"/run_multiple_benchmarks/ || exit 1
-	./run_main_benchmarks.sh "$1" "" "" "" "${ONLY_READS}" "" 2 "${OUTPUT}" 2>&1 |\
+	./run_main_benchmarks.sh "$1" "" "" "" "${ONLY_READS}" "" 5 "${OUTPUT}" 2>&1 |\
```
Comment (roxell) [inline automated/linux/ssuite/run-bench.sh]: Would it be possible to add number of repetitions as a variable that can be changed from the ssuite-bench.yaml and add a default value "5" ?

**Comment by Algodev-github** [comment]
> Just let me know which one to close and I'll do that.

Close #123 please.

File: automated/linux/ssuite/run-bench.sh (hunk 1 above)
Comment (Algodev-github) [reply automated/linux/ssuite/run-bench.sh]: Absolutely! Just let me first understand what Milosz is trying to explain to me on the multiple-value issue.

**Comment by mwasilew** [comment]
Closing this PR. Let's move discussion to #124

**Pull request closed without merging**